data/campaigns/Heir_To_The_Throne/utils/abilities.cfg

```diff
@@ -107,6 +107,13 @@ _"This unit's grasp of melee tactics allows adjacent allies to strike the first
                 {INITIATIVE_OBJECT x,y=$this_item.x,$this_item.y}
             [/do]
         [/foreach]
+        [lua] # Can't see why {INITIATIVE_OBJECT} leaves the last unit selected.
+            code=<<
+            wesnoth.deselect_hex()
+            -- Take THAT, you stupid bug! **SMASH**
+            -- (Gotta love the 10# sledge when you really should use a tap hammer!)
+            >>
+        [/lua]
 
```

Comment:
HA! There it is. All I need to do is remove the description from Li'sar's objects. There is no need for them that I can see, anyway since the player never even knows they're there.

	-- Default to silent if object has no description
	local silent = cfg.silent
	if silent == nil then silent = (text:len() == 0) end

	if not silent then
		wml_actions.redraw{}
		local name = tostring(cfg.name or "")
		wesnoth.show_popup_dialog(name, text, cfg.image)
	end

Gimme a while to put together a fix, test and push and I think this PR will _**finally**_ have all the issues fixed.